src/udev/udevd.c

```diff
@@ -1554,9 +1554,9 @@ static int parse_argv(int argc, char *argv[]) {
                                 log_warning("Invalid --children-max ignored: %s", optarg);
                         break;
                 case 'e':
-                        r = safe_atoi(optarg, &arg_exec_delay);
+                        r = parse_sec(optarg, &arg_exec_delay_usec);
                         if (r < 0)
-                                log_warning("Invalid --exec-delay ignored: %s", optarg);
+                                log_warning_errno(r, "Failed to parse --exec-delay value '%s', ignoring: %m", optarg);
```

Comment:
while you are it, I think we should write this as `--exec-delay=` rather than just `--exec-delay`. I think it's a good idea to systematically suffix cmdline switches (in fact all kinds of config switches) that take an argument with `=`, to clarify that they do